python/jittor/compiler.py

```diff
@@ -966,3 +971,5 @@ def convert_nvcc_flags(nvcc_flags):
 flags.gdb_path = gdb_path
 flags.addr2line_path = addr2line_path
 flags.has_pybt = has_pybt
+
+core.set_lock_path(lock.lock_path)
```

Comment:
I cannot find lock has lock_path attribute.

Reply:
https://github.com/Jittor/jittor/pull/59/files#diff-c76d8d8597aab86595040b4479f5a950R56
it is here